instrumentation/r2dbc-1.0/library/src/main/java/io/opentelemetry/instrumentation/r2dbc/v1_0/internal/DbExecution.java

```diff
@@ -94,10 +94,18 @@ public String getConnectionString() {
     return connectionString;
   }
 
+  /**
+   * @deprecated use {@link #getDbQueryText()} instead
+   */
+  @Deprecated
   public String getRawStatement() {
     return rawStatement;
   }
```

Comment:
can delete this method straight away since this class is only used by Java agent